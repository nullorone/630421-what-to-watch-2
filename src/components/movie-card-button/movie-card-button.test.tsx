import * as React from "react";
import renderer from "react-test-renderer";
import MovieCardButton from "./movie-card-button";

describe(`Make snapshot`, () => {
  const initProps = {
    iconName: `keks`,
    classModifier: `success`,
    text: `Javascript`,
  };

  it(`Get snapshot component`, () => {
    const tree = renderer
      .create(
          <MovieCardButton {...initProps}/>
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
