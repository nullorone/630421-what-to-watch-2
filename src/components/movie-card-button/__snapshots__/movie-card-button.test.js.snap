// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Make snapshot Get snapshot component 1`] = `
<button
  className="btn btn--success movie-card__button"
  type="button"
>
  <svg
    height="19"
    viewBox="0 0 19 19"
    width="19"
  >
    <use
      xlinkHref="#keks"
    />
  </svg>
  <span>
    Javascript
  </span>
</button>
`;
